Aftonbladet.js

```diff
@@ -0,0 +1,310 @@
+{
+	"translatorID": "235c5c47-0c04-4d77-9afe-e22265a017a9",
+	"label": "Aftonbladet",
+	"creator": "Jonatan Svensson Glad",
+	"target": "^https?://((ww(w|c)|tv)\\.)?aftonbladet\\.se",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-08-10 20:38:00"
+}
+
+/*
+	Aftonbladet Translator - Parses Aftonbladet articles and creates Zotero-based metadata.
+	Copyright (C) 2016 Jonatan Svensson Glad
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+*/
+
+	function detectWeb(doc, url) {
+		if (url.indexOf('/article')>-1) {
+			return "newspaperArticle";
+		} else if (getSearchResults(doc, true)) {
+			return "multiple";
+		}
+	}
+	
+	
+	function getSearchResults(doc, checkOnly) {
+		var items = {};
+		var found = false;
+		var rows = ZU.xpath(doc, '//a[contains(@href, "/article")]');
+		for (var i=0; i<rows.length; i++) {
+			var href = rows[i].href;
+			var title = ZU.xpathText(rows[i], './/h2');
+			if (!href || !title) continue;
+			if (checkOnly) return true;
+			found = true;
+			items[href] = title;
+		}
+		return found ? items : false;
+	}
+	
+	
+	function doWeb(doc, url) {
+		if (detectWeb(doc, url) == "multiple") {
+			Zotero.selectItems(getSearchResults(doc, false), function (items) {
+				if (!items) {
+					return true;
+				}
+				var articles = new Array();
+				for (var i in items) {
+					articles.push(i);
+				}
+				ZU.processDocuments(articles, scrape);
+			});
+		} else {
+			scrape(doc, url);
+		}
+	}
+	
+	function scrape(doc) {
+		var newArticle = new Zotero.Item('newspaperArticle');
+		newArticle.title = ZU.xpathText(doc, "//h1").replace("\n", " ");	
+		newArticle.date = ZU.xpathText(doc, '//time[@pubdate]') || ZU.xpathText(doc, '//div[@class="channel-info-metadata abLabelThin"]/span/text()'); 	// TODO:  Fix date for tv.aftonbaldet.se (xpath looks fragile and the result cannot be normalized)
+		var authors = ZU.xpath(doc, '//address[contains(@class, "abByline")]/div[contains(@class, "abAuthor")]');
+		for (var i=0; i<authors.length; i++) {
+			newArticle.creators.push(ZU.cleanAuthor(authors[i].textContent, "author"));
+		} //TODO Fix authors for /debatt/ e.g. http://www.aftonbladet.se/debatt/article23309432.ab
```

Comment:
Issue is that most of the /debatt/ doesn't have structurized authors, but it is  instead in the article text with e.g. bold/strong...I can't figure this out, since different /debatt/ is sructured differently...